src/test/scala/ch/epfl/pop/tests/pubsub/PubSubTest.scala

```diff
@@ -256,7 +256,7 @@ class PubSubTest extends FunSuite {
     sendAndVerify(l ::: messages)
   }
 
-  test("Create multiple LAOs, subscribe to their channel and publish multiple messages") {
+  ignore("Create multiple LAOs, subscribe to their channel and publish multiple messages") {
```

Comment:
Strange that it show a change, the test is already ignored on the be2-scala branch. Btw all the ignored tests are fixed in #186.